params/config_extra.go

```diff
@@ -59,10 +59,10 @@ func SetEthUpgrades(c *ChainConfig) {
 }
 
 func GetExtra(c *ChainConfig) *extras.ChainConfig {
-	ex := payloads.FromChainConfig(c)
+	ex := payloads.ChainConfig.Get(c)
```

Comment:
Didn't I review this in another PR? This suggests (along with the `nativeasset` change) that something's wrong in the commit history here.